### PR TITLE
fix #1713

### LINKED ISSUE #1713: Empty product information block is displayed if a product hasn't description
**Description**:
Empty product information block is displayed if a product hasn't description

**Expected behavior**:
Don’t show product information block if a product hasn't description
image

**To Reproduce**:
1. Open product page without description - https://demo.scandipwa.com/leitz-10120025-ring-binder-2-x-333826.html
2. Scroll down to product description

**Actual result**:
Empty block (product information) is displayed
image

### PR DESCRIPTION
**Pull request by ahmad0salah**

### CODE REVIEW

File: src/app/component/ProductInformation/ProductInformation.component.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ export class ProductInformation extends PureComponent {
 
         return (
             <ExpandableContent
-              // show placeholder if the details are not loaded
+                // show placeholder if the details are not loaded
```
Comment (yeegor) [inline src/app/component/ProductInformation/ProductInformation.component.js:80]: Unexpected :D

File: src/app/component/ProductInformation/ProductInformation.component.js [hunk 2]
```diff
@@ -87,6 +87,29 @@ export class ProductInformation extends PureComponent {
...
+        if (!htmlString || htmlString.trim() === '') {
+            return true;
+        }
+
+        const parser = new DOMParser();
+        const document = parser.parseFromString(htmlString.trim(), 'text/html');
+
+        if (document.body.children.length === 0) {
+            return false;
+        }
```
Comment (yeegor) [inline src/app/component/ProductInformation/ProductInformation.component.js:100]: Why ignore invalid HTML here? Let's stick to the behavior we see in browser and allow writing plain text.
image

File: src/app/component/ProductInformation/ProductInformation.component.js [hunk 3]
```diff
@@ -87,6 +87,29 @@ export class ProductInformation extends PureComponent {
...
+
+        // eslint-disable-next-line fp/no-let
+        let isWhiteSpace = true;
+        Array.from(document.body.children).forEach((element) => {
+            if (element.innerText !== '') {
+                isWhiteSpace = false;
+            }
+        });
+
+        return isWhiteSpace;
```
Comment (yeegor) [inline src/app/component/ProductInformation/ProductInformation.component.js:110]: 1. This logic is not intuitive, please comment on it.
2. Avoid using let, it is prohibited by the ESLint rules. 
3. The same can be written by using `Array.prototype.find` method, please avoid mutating the scope inside of loops.

**Review by yeegor — CHANGES_REQUESTED** [review]

File: src/app/component/ProductInformation/ProductInformation.component.js (hunk 2 above)
Comment (ahmad0salah) [reply src/app/component/ProductInformation/ProductInformation.component.js]: This condition ensures plain text is not ignored. 
*the function is supposed to return `false` if the string has content and will **not** render white space.

**Review by yeegor — APPROVED** [review]
Good! Clear and verbose. Nice job!

**Pull request merged**